Dockerfile

```diff
@@ -31,6 +31,14 @@ RUN [ -s /home/extractor/requirements.txt ] && \
     (echo 'No python modules to install' && \
      rm /home/extractor/requirements.txt)
 
+RUN (echo "installing sudo" && \
```

Comment:
You don't need sudo, by default the user account used by Docker when building an image is `root`. Can remove this and the sudo's below.